arm_demo_node:
  ros__parameters:
    planner: RRTstar
    step_size: 0.05 # distance step to check connection between nodes
    expansion_size: 0.5 # max distance of tree expansion
    bias: 0.5 # bias to expand towards goal
    solve_time: 5.0 # available time to find solution
    collision_radius: 0.01 # radius for collision checking with the environment​
    goal_tolerance: 0.01 # error tolerated by the end point of the path to the goal​
